src/components/common.js

```diff
@@ -228,3 +228,31 @@ export const LargeFadeBox = ({ children }) => {
     }, [children])
   ])
 }
+
+const viewToggleStyles = {
+  toolbarContainer: {
+    flex: 'none', display: 'flex', alignItems: 'flex-end',
+    color: colors.blue[0], padding: '0 2.25rem'
+  },
+  toolbarButton: active => ({
+    display: 'flex', justifyContent: 'center', alignItems: 'center',
+    borderRadius: 3, border: `1px solid ${colors.blue[0]}`,
+    height: '2.25rem', padding: '0 .75rem',
+    color: colors.blue[0],
+    backgroundColor: active ? colors.blue[4] : 'white',
+    fontWeight: active ? 'bold' : 'normal'
+  })
+}
+
+export const viewToggleButtons = (listView, setListView) => {
```

Comment:
TOL: Good abstraction, and these seem to be the right params to pass in.

With the signature as `(listView, setView)`, this is effectively a helper function that returns a snippet of DOM. Totally reasonable. Another option would be to make the signature `({ listView, setView })`, which would make it a helper _component_ (since it conforms to the `props => DOM` signature that React expects for functional components). That means you could e.g. render it with `h()` instead of calling it like a function.

We have both styles in our code, so it's a judgement call. Why choose the latter? If you end up needing to add more params, e.g. a `style`, it would probably be cleaner to roll everything up into a props object rather than adding more positional args.